docs/messaging/messaging-spans.md

```diff
@@ -180,10 +180,10 @@ The span name SHOULD be set to the message destination name and the operation be
 The destination name SHOULD only be used for the span name if it is known to be of low cardinality (cf. [general span name guidelines](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.26.0/specification/trace/api.md#span)).
 This can be assumed if it is statically derived from application code or configuration.
 Wherever possible, the real destination names after resolving logical or aliased names SHOULD be used.
-If the destination name is dynamic, such as a [conversation ID](#conversations) or a value obtained from a `Reply-To` header, it SHOULD NOT be used for the span name.
+If the destination name is dynamic, such as a [conversation ID](../attributes-registry/messaging.md) or a value obtained from a `Reply-To` header, it SHOULD NOT be used for the span name.
 In these cases, an artificial destination name that best expresses the destination, or a generic, static fallback like `"(anonymous)"` for [anonymous destinations](#temporary-and-anonymous-destinations) SHOULD be used instead.
 
-The values allowed for `<operation name>` are defined in the section [Operation names](#operation-names) below.
+The values allowed for `<operation name>` are defined in the well-known values section for the [`messaging.operation`](../attributes-registry/messaging.md) attribute.
```

Comment:
I think this should be reverted too (as now the "Operation names" section is kept).